src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -537,6 +537,7 @@ protected function getEnvParameters()
         $parameters = array();
         foreach ($_SERVER as $key => $value) {
             if (0 === strpos($key, 'SYMFONY__')) {
+                @trigger_error(sprintf('The special environment variables that start with SYMFONY__ (such as "%s") are deprecated as of 3.3 and will be removed in 4.0. Use instead the %env()% syntax to get the value of environment variables in configuration files.', $key), E_USER_DEPRECATED);
```

Comment:
` and will be removed in 4.0`

We cant really remove user env vars ^^ i'd go with `and will not be supported anymore in 4.0`.